.github/issue_template.md

```diff
@@ -1,3 +1,10 @@
+<!-- Title should describe the feature or problem from the user's perspective, and/or be directly actionable for someone taking on the issue -->
```

Comment:
Does this autofill the title? If so, then line 10 below should be redundant

Reply:
Sadly no.